libnetwork/libnetwork_linux_test.go

```diff
@@ -1790,12 +1791,25 @@ func reexecSetKey(key string, containerID string, controllerID string) error {
 	return cmd.Run()
 }
 
+// Strip comments that end in a newline (a comment with no newline at the end
+// of the file will not be stripped).
+var stripCommentsRE = regexp.MustCompile(`(?m)^#.*\n`)
+
+func assertContentEqual(t *testing.T, actual, expected []byte) {
+	t.Helper()
+	actual = stripCommentsRE.ReplaceAllLiteral(actual, []byte{})
+	if !bytes.Equal(actual, expected) {
+		t.Fatalf("Got:\n%s\nExpected:\n%s", actual, expected)
```

Comment:
Even if you're avoiding gotest.tools in this file for a reason, you could still call https://pkg.go.dev/github.com/google/go-cmp/cmp#Diff directly to be nice to whoever has to diagnose test failures.